### DISCUSSION TITLE
data-tauri-drag-region on another mouse event (not left click)

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#8220] [Q&A]
`data-tauri-drag-region` attribute allows defining a custom draggable region.
What if I want to drag only when right mouse or middle button is pressed instead of left click?
Is there an event associated with this?

**Comment by FabianLars** [comment]
The default handler can't do that but if you copy this code: https://github.com/tauri-apps/tauri/blob/1.x/core/tauri/scripts/core.js#L147 you can change the e.button check to whatever you want.
